shared/src/business/entities/cases/Case.js

```diff
@@ -2267,8 +2267,20 @@ const caseHasServedDocketEntries = rawCase => {
   return !!rawCase.docketEntries.some(docketEntry => isServed(docketEntry));
 };
 
+/**
+ * determines if the case is in a state where documents can be served
+ *
+ * @param {string} caseCaption the original case caption
+ * @returns {string} caseTitle the case caption with the postfix removed
+ */
+
+const canAllowDocumentServiceForCase = rawCase => {
```

Comment:
While working on #7643 we created a function similar to this -- and I'm wondering if it should be used instead, or if its conditions are entirely different:
https://github.com/flexion/ef-cms/blob/7643-sqs-integration/shared/src/business/entities/cases/Case.js#L1193
(the above branch is part of code that is in a long-running story)
As you can see, our code also checks if case has been closed inside the last six months.  I'm not certain whether this is the way to do it, or if these require different logic.  Just wanted to put this out there.

Reply:
I don't think that was the intent of this particular story. Per the PO, we want to disallow document service on a case that is in NEW status. This method is _currently_ only checking for that. I left it as a method instead of just checking its status everywhere in case this logic changed, at least tit would be abstracted. I'm wondering if I can just use the `isSttatusNew` property instead of exposing the status on Public case. That would give us what we want and we wouldn't have to make any changes to the entity, API response, or associated tests.